uphold-integration/src/main/res/values/strings-uphold.xml

```diff
@@ -30,4 +30,14 @@
     <string name="uphold_api_error_identity">Your Uphold account isn’t verified yet. Funds cannot be withdrawn.</string>
     <string name="uphold_api_error_403_description">Your Uphold account has been flagged for review.\n\nUphold has temporarily restricted your account and you won’t be able to withdraw funds.\n\nUphold will be in touch to help you sort this out quickly.</string>
     <string name="uphold_api_error_400_description">Uphold funds are locked for 24 hours.\n\nYou will have sufficient funds for this transfer by %s.</string>
+    <string name="uphold_api_error_400_locked">Uphold funds (%s %s) are locked for up to 65 days.\n\nYou will have sufficient funds for this transfer by %s.</string>
```

Comment:
For multiple arguments in strings, it might be better to declare them as `%1$s`, `%2$s`, etc, otherwise there will be warnings.
https://developer.android.com/guide/topics/resources/string-resource#formatting-strings